requirements-dev.txt

```diff
@@ -6,3 +6,4 @@ autopep8==1.3.3
 coverage==4.3.4
 codecov>=1.5.1
 git+https://github.com/hayd/pep8radius.git
+configobj==5.0.6
```

Comment:
I know that these are just dev requirements, but let's not pin to a specific version here, because we want the application (`pgcli`) to be able to do the pinning.